README.md

```diff
@@ -18,6 +18,19 @@ local debugging are supported. Snapshot Debugger only works with the
 [Node.js](https://github.com/googleapis/cloud-debug-nodejs) agent for preview.
 
 
+### Upcoming Features
+
+* Java agent support
+* Python 3 agent support
+* App Engine support for Java, Python 3, and Node.js
+
+Note:
+
+*  All upcoming features will be implemented before the
```

Comment:
I would leave this promise out just in case there is a delay.

Reply:
Entirely fair concern, however this promise has already been implied in prior communication so I do not believe that this statement alters anything.